quantmsio/core/maxquant.py

```
@@ -0,0 +1,304 @@
+            return None
+        else:
+            key = match.group(1)
+            if key in self.mods_map:
+                return f"({self.mods_map[key]})"
+            else:
+                return None
+
+    def get_mods_map(self, report_path):
+        if os.stat(report_path).st_size == 0:
+            raise ValueError("File is empty")
+        f = codecs.open(report_path, "r", "utf-8")
```

Ruff

Use a context manager for opening files

(SIM115)